src/platform/rock/rock4.c

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
Can you fix the source with proper styling like removing these bogus lines, keep indentation consistent etc.